src/typescript/sdk/src/indexer-v2/types/index.ts

```diff
@@ -596,6 +601,9 @@ export const toPriceFeed = (data: DatabaseJsonType["price_feed"]) => {
   };
 };
 
+export const toRandomSymbolsRPCResponse = (data: DatabaseJsonType["random_symbols"]) =>
+  symbolBytesToEmojis("0" + data.emojis.substring(1));
```

Comment:
What exactly is this doing? Is this replacing the `\x` from the beginning of the string with `0x`? Let's put that in a comment, otherwise this is a bit confusing